src/AmbientSounds.Uwp/Services/MixMediaPlayerService.cs

```diff
@@ -143,28 +145,28 @@ public async Task ToggleSoundAsync(Sound s, bool keepPaused = false, string pare
                 return;
             }
 
-            if (IsSoundPlaying(s.Id))
+            if (IsSoundPlaying(s!.Id))
```

Comment:
is `!` really needed here? The if block above already ensure that `s` will not be null here.

Reply:
> "The if block above already ensure that s will not be null here."

That is 100% correct, but unfortunately the compiler doesn't know that on UWP 😢
The reason is that the BCL that UWP uses doesn't have any nullability annotations, so `string.IsNullOrWhiteSpace` cannot inform the nullable flow analysis that the compiler is doing, so it can't know that `s` cannot in fact ever be `null` at this point. So here I'm adding that `!` to suppress the warning (see image below without it), and then when/if we will get .NET 5/6 on UWP in the future (or in case you ported the app to WinUI 3 with .NET 5), then we could of course remove the `!` because that API we're using would already be properly annotated in that case 🙂

Screenshot with the issue on UWP without suppression:

![image](https://user-images.githubusercontent.com/10199417/115895488-c68be180-a45a-11eb-86e9-545bf5fdb065.png)